test/e2e/log_size_monitoring.go

```diff
@@ -212,9 +215,11 @@ func (g *LogSizeGatherer) Work() bool {
 	)
 	if err != nil {
 		Logf("Error while trying to SSH to %v, skipping probe. Error: %v", workItem.ip, err)
+		workItem.backoffMultiplier *= 2
```

Comment:
For now yes, but I can imagine that in the future we can have tests that last e.g. 24 hours or sth like that.
So I think it makes sense to add an upper bound (it will no hurt).